### ISSUE TITLE
Support backing up Settings data in AdClear

### ISSUE THREAD

**Issue by DustyWMorgan** [SEVENNetworks/AdClear#74]
Several users have requested backing up AdClear data noting that it is annoying when reinstalling AdClear they are required to toggle apps, settings, filters, etc. We should support backing up this data to make reinstall easier.

**Comment by rossbott** [comment]
I agree, this makes particular sense given the increased number of features in AdClear.



Sent from my T-Mobile 4G LTE Device


-------- Original message --------
